src/App.tsx

```diff
@@ -0,0 +1,30 @@
+import { ReactElement, useState } from 'react';
+
+import './App.css';
+import { Counter } from './Counter/Counter.tsx';
+import { Header } from './Header/Header.tsx';
+import { GenreSelect, SearchForm } from './components';
+
+function App(): ReactElement {
+	const [count, setCount] = useState(0);
+	const [searchTerm, setSearchTerm] = useState('');
+	const [selectedGenre, setGenre] = useState<string | null>('All');
+	const [genres] = useState(['All', 'Documentary', 'Comedy', 'Horror', 'Crime']);
```

Comment:
1) It`s better to keep such date in a separate place (maybe some config file)
2) You use `useState` hook but do not use setter method. In the case you don't need setter, you can use usual variable instead of hook

Reply:
The data could be in a separate config file and I could not use the useState hook for genres, but we will probably add the back-end so the hardcoded data may be gone later. Thank you